src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -2010,7 +2010,7 @@ namespace
             auto allIVs = ValueHandle::makeIndexHandles(vArg.rank());
             auto pAllIVs = makeHandlePointers(allIVs);
             auto steps = vArg.getSteps();
-            SmallVector<ValueHandle, 8> nonRedIVs;
+            SmallVector<ValueHandle, 8> nonRedIVs, tempIVs;
```

Comment:
Single decl per line, no commas, or is this ok here because it is in mlir?
```suggestion
            SmallVector<ValueHandle, 8> nonRedIVs;
            SmallVector<ValueHandle, 8> tempIVs;
```

Reply:
Borrowed from another PR, but I fixed it there and here.